dg/templates/social_website/collections.html

```diff
@@ -3,7 +3,32 @@
     {% include "includes/partials/search.html" %}
     
     <!-- Main page content -->
-    
+    <div class="connectorModal">
+        <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLabel">Enter you email to get data</h5>
```

Comment:
Change the string to "Enter a valid email to get FarmStack installation instructions"